tests/v2/validation/rbac/rbac_additional_test.go

```diff
@@ -60,116 +52,131 @@ func (rb *RBACAdditionalTestSuite) SetupSuite() {
 	require.NoError(rb.T(), err)
 }
 
-func (rb *RBACAdditionalTestSuite) ValidateAddStdUserAsProjectOwner() {
+func (rb *RBACAdditionalTestSuite) TestRbacP1ClusterOwner() {
+	subSession := rb.session.NewSession()
+	defer subSession.Cleanup()
 
-	createProjectAsCO, err := createProject(rb.client, rb.cluster.ID)
+	clusterAdmin, standardUserClient, err := setupUser(rb.client, standardUser)
```

Comment:
I don't think standardUser is defined anywhere yet?

Reply:
Standard user is a constant defined in rbac.go. this is what a standard user looks like:
```	
standardUser                  = "user"
```